### PR TITLE
89/Server - Fork Applicatif

### PR DESCRIPTION
**Pull request by LucasTesnier**
Implement the Fork command in the applicatif side.

Fix some missing free.

### CODE REVIEW

**Review by Cavonstavant — APPROVED** [review]
:+1:

File: src/zappy_server_src/core/include/entity/entity.h [hunk 1]
```diff
@@ -91,7 +92,7 @@ void entity_wrapper_add_player(entity_wrapper_t *wrapper, entity_t *player);
 /// \param wrapper the entity wrapper to add the egg to
 /// \param position the egg position
 /// \param team_name the egg team_name
-void entity_wrapper_create_egg(entity_wrapper_t *wrapper, position_t pos,
+uuid_t *entity_wrapper_create_egg(entity_wrapper_t *wrapper, position_t pos,
```
Comment (osvegn) [inline src/zappy_server_src/core/include/entity/entity.h:95]: missing return value in documentation

**Review by osvegn — CHANGES_REQUESTED** [review]
LOOK GOOD, BUT NEED ONE MINOR CHANGE

**Review by 31Nathan — APPROVED** [review]
LGTM!

**Pull request merged**